orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/support/DeploymentStrategyParameterSupport.groovy

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2015 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.pipeline.support
+
+import com.netflix.spinnaker.orca.pipeline.model.Pipeline
+
+class DeploymentStrategyParameterSupport {
```

Comment:
But you'd still have to explicitly call the fn?
On Mon, Dec 7, 2015 at 5:03 PM Tomás Lin <notifications@github.com> wrote:

> In
> orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/support/DeploymentStrategyParameterSupport.groovy
> <https://github.com/spinnaker/orca/pull/677#discussion_r46884692>:
>
> > + * You may obtain a copy of the License at
> > + *
> > + *    http://www.apache.org/licenses/LICENSE-2.0
> > + *
> > + * Unless required by applicable law or agreed to in writing, software
> > + * distributed under the License is distributed on an "AS IS" BASIS,
> > + * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > + * See the License for the specific language governing permissions and
> > + * limitations under the License.
> > + */
> > +
> > +package com.netflix.spinnaker.orca.pipeline.support
> > +
> > +import com.netflix.spinnaker.orca.pipeline.model.Pipeline
> > +
> > +class DeploymentStrategyParameterSupport {
>
> Not sure it's the right approach as there are some stages where you don't
> want this to happen
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spinnaker/orca/pull/677/files#r46884692>.
>